### DISCUSSION TITLE
Setting Menu jumps out

### DISCUSSION THREAD

**Discussion by Shahin-rmz** [Schneegans/Fly-Pie#345] [Fly-Pie Q&A]
Hello,
I am new to the Fly pie.
It boosts our productivity a lot. Thanks for forking and maintaining it.
I just have a problem with setting menu. It jumps out after some seconds.
Thanks

**Comment by Schneegans** [comment]
This sounds like #313. I currently have no solution for this issue, but you could add your system details to that issue (if it's really the same bug).
